src/openlcb/TractionCvSpace.cxxtest

```diff
@@ -62,9 +62,9 @@ protected:
         wait();
     }
 
-    size_t expected_feedback_key()
+    uint32_t expected_feedback_key()
     {
-        return reinterpret_cast<size_t>(&cv_space_);
+        return (size_t)(&cv_space_);
```

Comment:
Why not use uintptr_t

Reply:
here we need to guess what the TractionCvSpace will send as feedback key in order to fabricate  the appropriate returned object from the DCC driver.

Reply:
done